src/app/routes/regex/utils/index.js

```diff
@@ -5,6 +5,7 @@ const variantRegex = '/simp|/trad|/cyr|/lat';
 const articleLocalRegex = 'articles|erthyglau|sgeulachdan';
 const mediaIdRegex = '[a-z0-9]+';
 const mediaServiceIdRegex = 'bbc_[a-z]+_radio|bbc_[a-z]+_tv';
+const radioScheduleIdRegex = 'bbc_[a-z]+_radio';
```

Comment:
This might crash with `mediaServiceIdRegex` since it is exactly the same as the first condition on the `mediaServiceIdRegex`.

Reply:
It will clash but maybe we can switch the order in [routes](https://github.com/bbc/simorgh/blob/latest/src/app/routes/index.js#L49) to match radioSchedule first.

Reply:
Took a second look, https://github.com/bbc/simorgh/pull/5165/files#diff-15d3ba1f19c229252fd4ec5e6f504dd8R55 should cover for this.

Reply:
Opted to remove the added `radioScheduleDataPath` as the existing [getRadioAndTVRegex](https://github.com/bbc/simorgh/blob/a7f8b80dab7189a826b20cefb9c276a24279b44a/src/app/routes/regex/utils/index.js#L47) also caters to radioschedules paths